src/app/tests/suites/certification/Test_TC_S_2_3.yaml

```diff
@@ -20,99 +20,581 @@ PICS:
 
 config:
     nodeId: 0x12344321
-    cluster: "Basic Information"
-    endpoint: 0
+    cluster: "Scenes"
+    endpoint: 1
 
 tests:
+    - label: "Wait for the commissioned device to be retrieved"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "TH reads from the DUT the (0x0006) SceneTableSize attribute"
+      PICS: S.S.A0006
+      command: "readAttribute"
+      attribute: "SceneTableSize"
+      response:
+          values:
+              - name: "SceneTableSize"
+                saveAs: maxScenes
+          constraints:
+              type: uint16
+
     - label:
-          "preparation step for using commands from Groups cluster: TH generates
-          fabric-unique GroupID, GroupName, random key, EpochKey0 and
-          GroupKeySetID. TH sets GroupKeySecurityPolicy = TrustFirst (1)
-          GroupKeyMulticastPolicy = PerGroupID (0) EpochStartTime0 = 0"
-      verification: |
+          "preparation step for using commands from Groups cluster: Add KeySet"
+      cluster: "Group Key Management"
+      endpoint: 0
+      command: "KeySetWrite"
+      arguments:
+          values:
+              - name: "GroupKeySet"
+                value:
+                    {
+                        GroupKeySetID: 0x01a1,
+                        GroupKeySecurityPolicy: 1,
```

Comment:
This is going to fail.  TrustFirst is 0, not 1.  Does the test plan need fixing?

Reply:
Created [issue ](https://github.com/CHIP-Specifications/chip-test-plans/issues/2746) and changed to 0.